tap_salesforce/__init__.py

```diff
@@ -51,19 +51,21 @@ def main_impl():
     stream = Stream(args.state)
     sync_fields(sf, stream)
 
-    for stream_id, fields, replication_key in sf.get_tables():
+    for table, fields, replication_key in sf.get_tables():
         if not fields:
             LOGGER.info(
-                f"skipping stream {stream_id} since it does not exist on this account"
+                f"skipping stream {table.name} since it does not exist on this account"
```

Comment:
This message seems a little misleading: wouldn't this indicate, that there are simply no fields for the table?

I see that table existence is handled in the `get_tables` function.

Reply:
The code trying to get fields for a table to check if the table exist or not. I guess we need to call `/services/data/{self._API_VERSION}/sobjects` endpoint to check if the table exist.

Reply:
But that need some refactor to make the logic correct. I will leave it as it is for now.